.github/workflows/linux-flatpak.yml

```diff
@@ -0,0 +1,85 @@
+name: Linux (Flatpak)
+
+on:
+  workflow_dispatch:
+    inputs:
+      myCommit:
+        description: Commit SHA1
+        required: false
+        default: ''
+        type: string
+      releaseTag:
+        description: Release tag
+        required: false
+        default: ''
+        type: string
+  workflow_call:
+    inputs:
+      myCommit:
+        description: Commit SHA1
+        required: false
+        default: ''
+        type: string
+      releaseTag:
+        description: Release tag
+        required: false
+        default: ''
+        type: string
+  push: null
+  pull_request: null
+  release:
+    types:
+      - published
+
+jobs:
+  build-flatpak-bundle:
+    name: "Build flatpak bundle"
+    runs-on: ubuntu-latest
+    container:
+      image: bilelmoussaoui/flatpak-github-actions:freedesktop-23.08
```

Comment:
Why it's 23 and not some other version? That's a noob question, I have no idea what the consequences of this choice are.

Reply:
The version just determines which Flatpak runtimes/SDKs are preinstalled to the container. However, given that the actual manifest uses 24.08 I'll need to update that here...